e2e/test/scenarios/admin/databases/database-exceptions.cy.spec.js

```diff
@@ -0,0 +1,92 @@
+import { restore, typeAndBlurUsingLabel, isEE } from "e2e/support/helpers";
+
+describe("scenarios > admin > databases > exceptions", () => {
```

Comment:
Maybe it's not such a bad idea to have at least one `exceptions` spec per group/folder where we can handle edge cases.